README.md

```diff
@@ -55,7 +55,7 @@ There are several ways to install Terminus, depending on your use case:
 1. Make the Terminus file exectable. The example below assumes the same installation path as above:
 
     ```bash
-    chmod +X ~/bin/terminus
+    chmod +x ~/.bin/terminus
```

Comment:
Putting a `.` in the file path will make this not go to the same place as above.

Reply:
Yup. Feel free to edit it.